lib/minfraud/validates.rb

```diff
@@ -24,6 +24,16 @@ def validate_md5(field, value)
       end
     end
 
+    def validate_uuid(field, value)
+      return if !value
+
+      uuid_regex = /\A[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\z/i
```

Comment:
I'm not sure if we want to use a regex (or this specific regex) for this or not. I just noticed the python library explicitly tests that all of these work:
```
            "12345678-1234-1234-1234-123456789012",
            "1234-5678-1234-1234-1234-1234-5678-9012",
            "12345678901234567890123456789012",
```

Reply:
Huh, I am not sure why it does that. I could possibly see with hyphens and without, but as far as I know we always display the ID in the standard form. I guess we should check what the web service does though and match that  so that we don't introduce a breaking change.